sdk/containerservice/azure-mgmt-containerservice/azure/mgmt/containerservice/v2019_11_01/operations/_managed_clusters_operations.py

```diff
@@ -447,6 +447,71 @@ def list_cluster_user_credentials(
         return deserialized
     list_cluster_user_credentials.metadata = {'url': '/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ContainerService/managedClusters/{resourceName}/listClusterUserCredential'}
 
+    def list_cluster_monitoring_user_credentials(
```

Comment:
@julienstroheker any idea why was this added now?  As far as I can see monitoring user credentials are only found in 2019-10-01
2019-11-01
2020-01-01